ftl/core/preferences.ftl

```diff
@@ -19,7 +19,8 @@ preferences-preferences = Preferences
 preferences-scheduling = Scheduling
 preferences-show-learning-cards-with-larger-steps = Show learning cards with larger steps before reviews
 preferences-show-next-review-time-above-answer = Show next review time above answer buttons
-preferences-spacebar-rates-card = Spacebar also answers card
+preferences-enter-key-rates-card = Press Spacebar/Enter to answer card
```

Comment:
This is technically more correct, but I wonder whether there are many users who actually prefer enter over space, and if not, whether the more succinct description would be better. The enter shortcut mainly exists for legacy - at one point in the past, enter would reveal the answer but not rate the card.